dubbo-samples-merge/dubbo-samples-merge-provider1/src/main/resources/spring/merge-provider.xml

```diff
@@ -25,7 +25,7 @@
 
     <dubbo:application name="merge-provider-1"/>
 
-    <dubbo:registry address="zookeeper://${zookeeper.address:127.0.0.1}:2181"/>
+    <dubbo:registry address="zookeeper://${zookeeper.address:192.168.0.156}:2181"/>
```

Comment:
pls not change this address, which will cause address problem for other users

Reply:
ok, the address of zk had been reverted.